js-libs/telemetry/TelemetryV3Manager.js

```diff
@@ -203,7 +203,7 @@ TelemetryV3Manager = Class.extend({
         }
         var responseData = {
             target: target,
-            type: optionTag,
+            type: optionTag || data.type,
```

Comment:
This should happen before the switch case.  Check the telemetry V3 event of Reponse
https://github.com/ekstep/Common-Design/wiki/Telemetry-v3#response

"type" value should be always mentioned in the switch case for (MCQ, FTB etc.)
@pallakartheekreddy